assets/css/menu.scss

```
@@ -30,5 +30,31 @@
 		margin-top: -1px;
 		position: relative;
 		top: 1px;
+		display: flex;
+		justify-content: space-between;
+		align-items: center;
+	}
+
+
+	/**
+	 * Adjust upsells styles
+	 */
+	.wpjm-addon-upsell__badge {
+		border-radius: 16px;
+		color: inherit;
+		font-size: 8px;
+		margin: 0;
+		padding: 4px 8px;
+		text-align: center;
+		height: fit-content;
+		width: fit-content;
+		display: block;
+		line-height: 8px;
+		vertical-align: middle;
+		box-sizing: border-box;
+		min-width: 16px;
+		border: 1px solid currentColor;
+		font-weight: 600;
+		z-index: 26;
```

Just a little cleanup, some of these are not really needed now.

```suggestion
		border-radius: 16px;
		color: inherit;
		font-size: 8px;
		margin: 0;
		padding: 4px 6px;
		line-height: 8px;
		min-width: 16px;
		border: 1px solid currentColor;
		font-weight: 600;
		z-index: 1;
```

---

Done in af116f5. Thanks!  (I kept the `padding` as is, though, is that a problem?)